### PR TITLE
Add resources for VN-216 ticket

### PR DESCRIPTION
**Pull request by votuanphong1983**
Add resources for VN-216 ticket

### CODE REVIEW

File: cs/index.json [hunk 1]
```diff
@@ -99,6 +99,7 @@
     "wattage_cost": "Wattage cost",
     "mining_electricity_consumption": "Mining electricity consumption of your GPUs/CPUs",
     "electric_cost": "Electricity cost for 1 kWh",
-    "estimate_again": "Estimate again"
+    "estimate_again": "Estimate again",
+    "auto_update_wattage" : "Auto update from Miner tool"
```
Comment (pandafox) [inline cs/index.json]: This key name doesn't really align with the text? What exactly are we trying to inform the user about?

Comment (votuanphong1983) [reply cs/index.json]: We have new checkbox on Wattage Setting called "Auto update from Miner tool" 
https://gitlab.com/mcnfreedom/vn-team/crypto.miner.client/uploads/38af60078d10d6c581a8fcbceb017ff8/image.png

That means `wattage` value will be updated automatically if Crypto Miner can get this info from Miner tool

Comment (pandafox) [reply cs/index.json]: Hum okay, I was going to suggest the following, but I guess there's not much difference.
use_detected_wattage: "Use wattage data obtained from Miner"

File: zh/index.json [hunk 2]
```diff
@@ -99,6 +99,7 @@
     "wattage_cost": "電費",
     "mining_electricity_consumption": "挖掘時的耗電量",
     "electric_cost": "電費 1 kWh",
-    "estimate_again": "重新預估"
+    "estimate_again": "重新預估",
+    "auto_update_wattage" : "Auto update from Miner tool"
```
Comment (pandafox) [inline zh/index.json]: "auto_update_wattage" : "自動使用挖掘時的電量資料"

File: cs/index.json [hunk 3]
```diff
@@ -99,6 +99,7 @@
     "wattage_cost": "Náklady",
     "mining_electricity_consumption": "Spotřeba el. energie vašich GPU/CPU při těžbě",
     "electric_cost": "Cena elektřiny za 1 kWh",
-    "estimate_again": "Odhadnout znovu"
+    "estimate_again": "Odhadnout znovu",
+    "use_detected_wattage" : "Use wattage data obtained from Miner"
```
Comment (czjohns) [inline cs/index.json]: "use_detected_wattage" : "Použít data o příkonu získaná minerem"

Comment (votuanphong1983) [reply cs/index.json]: @czjohns  Updated. Thanks

**Review by angels2it — APPROVED** [review]

**Pull request merged**